apps/src/lib/node/ledger/mod.rs

```diff
@@ -296,6 +208,88 @@ impl Shell {
                 current_chain_id, chain_id
             )));
         }
+        if self.last_state().is_none() {
```

Comment:
We could either skip this check or make it a fatal error as this shouldn't be called if there is a state